Data-Structures/src/main/java/utilities/MultiBracketValidation.java

```diff
@@ -0,0 +1,29 @@
+package utilities;
+
+public class MultiBracketValidation {
+
+  public static boolean multiBracketValidation(String input){
+    String[] openers = new String[]{"(", "[", "{"};
+    String[] closers = new String[]{")", "]", "}"};
+    String[] inputArr = input.split("");
+    boolean flag = false;
+    int index = 0;
+    if(inputArr.length % 2 == 1) {
```

Comment:
The `inputArr`'s length shouldn't be used to determine anything: `{odd}` and `[even]` should both return `true`